### DISCUSSION TITLE
[Bug?] Enabling the address toolbar causes the taskbar to get thicker

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#962] [General]
Steps:

1. Right click on the taskbar
2. Go to toolbars
3. Click Address

I am sure this is due to Microsoft making the search and address bars thicker in Explorer
Explorer Patcher build 22000.493.42.24
Windows 11 build 22000.527
This doesn't seem to be the most severe bug as most people don't use the toolbars

**Comment by valinet** [comment]
So it’s not a bug. It’s the same as in Windows 10. Same happens when you add a toolbar and set the icons to large.

When I’ll know how to fix it, I will fix it. Or, if anyone knows how to, in the mean time, they are more than welcome.
